src/current/_includes/v24.3/ldr/show-logical-replication-responses.md

```diff
@@ -0,0 +1,8 @@
+Field    | Response
```

Comment:
Is this missing the "description" field in LDR responses?

Reply:
yes it is, I am adding. Thanks for catching.